action.yml

```diff
@@ -77,9 +80,33 @@ runs:
         fi;
         echo "::set-output name=branch::$branch"
         echo "Using branch: $branch"
-        siteId=`jq -r '.searchspring.siteId' < package.json`
-        echo "::set-output name=siteId::$siteId"
-        echo "Using siteId: $siteId"
+        
+        secrets=`echo "${{ toJSON(inputs.secrets) }}"`;
```

Comment:
Is this being re-encoded toJSON a second time?

https://github.com/searchspring-implementations/snap.searchspring.io/blob/dennis/action-testing/.github/workflows/deploy.yml#L27

Reply:
Yup. However I figured out how to remove the double encoding!